skimage/transform/_geometric.py

```diff
@@ -816,10 +816,10 @@ class AffineTransform(ProjectiveTransform):
     Has the following form::
 
         X = a0*x + a1*y + a2 =
-          = sx*x*cos(rotation) - sy*y*sin(rotation + shear) + a2
+          = sx*x*cos(rotation) - sy*y*(tan(shear) * sin(rotation) + sin(rotation)) + a2
 
         Y = b0*x + b1*y + b2 =
-          = sx*x*sin(rotation) + sy*y*cos(rotation + shear) + b2
+          = sx*x*sin(rotation) - sy*y*(tan(shear) * sin(rotation) - cos(rotation)) + b2
```

Comment:
Do you perhaps have a good source to cite here? Perhaps one where you took your equations for $R_\theta$ and $S_{\phi_x}$ from? I'll see if I can find one myself to check this...

Reply:
I added Wikipedia's article as a reference. I think this is sufficient.

Reply:
Thanks. The [article](https://en.wikipedia.org/wiki/Affine_transformation#Image_transformation) is already an improvement. Unfortunately I can't find anything about how the shear angles relate to the matrix in there. I see the example but it tells me nothing how they arrived at the values in the matrix. But I haven't yet found a better source either.

Reply:
I found an article explaining shear transformation.

https://en.wikipedia.org/wiki/Shear_mapping

This says 
"the shear factor m is the cotangent of the shear angle $\phi$ between the former verticals and the x-axis."
Is this helpful?

Note that the skimage's version defines the angle $\phi'$ as the angle between the former verticals and the y-axis. So we should translate them with the equation $\phi' = \pi / 2 - \phi$, especially $m = \cot \phi = \tan \phi'$. And I used a negative angle ( $- \tan \phi'$ ) because the doc string of the `shear` parameter says the angle is counter-clockwise.

Reply:
> Note that the skimage's version defines the angle 
 as the angle between the former verticals and the y-axis.

Do you get that from the implementation? We should definitely add that to the docstring and documentation of the `shear` parameter.

Reply:
Sure, I added this one and the shear parameter description.

Reply:
> Do you get that from the implementation? 

Yes, I judged from the actual behavior and code.